Fix METS format metadata on DIP upload, refs #13401

replaceafill opened the pull request:
This converts the serialized `format` QubitProperty that stores METS
format metadata into separate QubitProperty objects with the name
expected by QubitObjectInformation.

A new migration has also been added to update and delete existing
properties.

mcantelon commented on lib/task/migrate/migrations/arMigration0188.class.php:
```diff
@@ -0,0 +1,84 @@
+
+/*
+ * Add separate properties for existing serialized DO format information.
+ * The names of the new properties match what is expected by QubitInformationObject
```
Nitpick: missing period.

mcantelon commented on lib/task/migrate/migrations/arMigration0188.class.php:
```diff
@@ -0,0 +1,84 @@
+    //   3. Delete the existing property
+    foreach (QubitProperty::get($criteria) as $property)
+    {
+      if (null !== $value = $property->getValue(array('sourceCulture'=>true)))
```
Nickpick: maybe add spaces around "->" in array for readability.

mcantelon reviewed (approved):
A couple of nitpicks that don't matter much, but otherwise looks good! 👍

replaceafill committed:
Fix METS format metadata on DIP upload, refs #13401

This converts the serialized `format` QubitProperty that stores METS
format metadata into separate QubitProperty objects with the name
expected by QubitObjectInformation.

A new migration has also been added to update and delete existing
properties.
